lib/plugins/aws/info/display.js

```diff
@@ -76,6 +76,8 @@ module.exports = {
       });
     }
 
+    message += 'Deploy completed: ' + new Date();
```

Comment:
I think you should use string interpolation syntax here.

Something like: `Deploy completed at ${new Date()}` should work.